src/convertToObject.js

```diff
@@ -1,18 +1,22 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesTab = sourceString.split(';').filter(el => el.trim() !== '');
+  const resultObj = {};
+
+  stylesTab.forEach(style => {
+    const [property, value] = style.split(':').map(el => el.trim());
```

Comment:
Same here fix chained methods alignment, and check if you can use more descriptive name for **el**